tutorial/chapter/3/index.md

```diff
@@ -125,32 +115,51 @@ function(x, y) {
 }
 ```
 
-JavaScript's usual relaxed rules around argument count can lead to extremely
-subtle and hard to detect bugs. It's potentially more flexible the JavaScript
-way, but much more error-prone.
+JavaScript's usual relaxed rules around argument count can lead to extremely subtle and hard to detect bugs. It's potentially more flexible the JavaScript way, but much more error-prone.
+
+Sometimes APIs are much easier if they can have varying parameter counts, so it is still possible in Squiggle. Using `...` in front of your last parameter causes it so collect all of the extra parameters in an array.
 
-Sometimes APIs are much easier if they can have varying parameter counts, so it
-is still possible in Squiggle. Using `...` in front of your last parameter
-causes it so collect all of the extra parameters in an array.
+### Multi-statement functions
+
+Functions may have multiple statements in them if you use this form with `do/end`:
+
+```squiggle
+let add = fn(x, y) do
+    console.log(x)
+    console.log(y)
+    x + y
+end
+```
+
+If you're defining a name function though, you should use the form `def/end`:
+
+```squiggle
+def add (x, y)
+    console.log(x)
+    console.log(y)
+    x + y
+end
+```
+
+Both of these forms automatically return the value of their last statement.
+
```

Comment:
what's the difference between the two forms.?
why are there two different froms?